typegraph/core/src/utils/fs_host.rs

```diff
@@ -214,3 +216,39 @@ pub fn hash_file(path: &Path) -> Result<(String, u32), String> {
 pub fn path_exists(path: &Path) -> Result<bool, String> {
     path_exists_host(&path.to_string_lossy())
 }
+
+pub fn is_glob(path: &str) -> bool {
+    let path = PathBuf::from(path);
+    if let Some(base_name) = path.file_name() {
```

Comment:
one issue here, only checking the file name for wild cards. will push the fix for this.